update documentation to match FortiOS device tracker updates (#20713)
home-assistant/home-assistant.github.io: source/_integrations/fortios.markdown

---

> title: "FortiOS"
> description: "Instructions on how to use Fortinet FortiOS to track devices in Home Assistant."
> ha_category: "Presence Detection"
> ha_iot_class: "Local Polling"
> ha_release: "0.97"

Reviewer: ```suggestion
title: FortiOS
description: Instructions on how to use Fortinet FortiOS to track devices in Home Assistant.
ha_category: Presence Detection
ha_iot_class: Local Polling
ha_release: 0.97
```

ha_domain: fortios
ha_platforms:
  - device_tracker
ha_codeowners:
  - '@kimfrellsen'
---

The FortiOS presence detection enables Home Assistant to track devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).

> The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).  
> The integration has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).  

Reviewer: ```suggestion
The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/). It has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).  
```

> The integration uses this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.

Reviewer: This is an implementation detail. Not relevant for the average end-user.
```suggestion
The minimum version supported is FortiOS 6.4.3.
```

> The latest update to the integration use this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.

Reviewer 2: ```suggestion
The integration uses this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.
```

All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and Wi-Fi devices, including devices detected by LLDP.

The integration is based on the Home Assistant `device_tracker` platform.

> ### Example Configuration:
> This example uses the FortiOS integration as device tracker, with API token and does not verify the SSL certificate.

Reviewer: ```suggestion
### Example Configuration:

This example uses the FortiOS integration as a device tracker, with an API token and does not verify the SSL certificate.

```

---
title: FortiOS
description: Instructions on how to use Fortinet FortiOS to track devices in Home Assistant.

> ha_category: Presence Detection

Reviewer: ```suggestion
ha_category: 
 - Presence Detection
```

  - Presence Detection
ha_release: 0.97
ha_iot_class: Local Polling

> ha_quality_scale: silver

Reviewer 2: Changing the quality scale requires updating the manifest.json in the core integration first.

Author: Thanks. I have removed the quality_scale. It is not of importance to me.
I do not understand the Netlify errors. They are pointing towards the "ha_iot_class: Local Polling" line and complaining of headers. I have tried to encapsulate them in " without any luck.

{% configuration %}
host:

>   description: Hostname or IP address of the FortiGate. Optionally portname can be added like this "10.10.10.10:443". Remember qoutes if portnumber is added.

Reviewer 2: ```suggestion
  description: Hostname or IP address of the FortiGate. Optionally port name can be added like this "10.10.10.10:443". Remember quotes if port number is added.
```

  required: true
  type: string
token:

>   description: See: [Generate an API token for FortiOS](https://docs.fortinet.com/document/forticonverter/6.2.0/online-help/866905/connect-fortigate-device-via-api-token) for how to create an API token. Remember this integration only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.

Reviewer 2: ```suggestion
  description: See: [Generate an API token for FortiOS](https://docs.fortinet.com/document/forticonverter/6.2.0/online-help/866905/connect-fortigate-device-via-api-token) for how to create an API token. This integration only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
```

  required: true
  type: string
verify_ssl:
  description: If the SSL certificate should be verified. In most home cases users do not have a verified certificate.
  required: false
  type: boolean

>   default: false  

Reviewer: ```suggestion
  default: false
```
